src/components/general/EntryVerify/index.tsx

```diff
@@ -108,22 +105,21 @@ const requestOptions: Requests<ComponentProps, Params> = {
 function EntryVerify(props: Props) {
     const {
         className,
-        hide,
-        verified: verifiedFromProps = false,
+        value = false,
         requests,
         title,
-        setEntryVerificationPending,
+        onPendingChange,
     } = props;
 
     const {
         setEntryVerificationRequest,
     } = requests;
 
-    const [verified, setVerificationStatus] = useState(verifiedFromProps);
+    const [verified, setVerificationStatus] = useState(value);
```

Comment:
Why use use a `verified` state when all it does is reflect on prop `value`

Reply:
This was already there, This component is very questionable and should be refactored. I wonder how it get merged.

Reply:
Yes, looks very bad!